v8.0.0.alpha/00-first_change.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
```

Comment:
Although this step still works, maybe we should combine overriding views, helpers, and view components into a later step.